src/arm/radxa_rock-3c.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * Author: Ken <ken@radxa.com>
+ * Copyright (c) Radxa Limited.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <mraa/common.h>
+#include <stdarg.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mman.h>
+#include "arm/radxa_rock3c.h"
+#include "common.h"
+
+#define DT_BASE "/proc/device-tree"
+
+#define PLATFORM_NAME_RADXA_ROCK_3_MODEL_C "Radxa ROCK3 Model C"
```

Comment:
```suggestion
#define PLATFORM_NAME_RADXA_ROCK_3C "Radxa ROCK3 Model C"
```
然后可以挪到`arm/radxa_rock-3c.h`里面